keyboards/linworks/fave84h/rules.mk

```
@@ -20,4 +20,4 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 RGB_MATRIX_ENABLE = yes
-RGB_MATRIX_DRIVER = WS2812
+RGB_MATRIX_DRIVER = WS2812
```

```suggestion
RGB_MATRIX_DRIVER = WS2812
```
All lines should end in a newline character.